spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -733,49 +830,56 @@ def set_recording(self, variable, new_state, sampling_interval=None,
             raise ConfigurationException("Variable {} is not supported".format(
                 variable))
 
-    def _get_buffered_sdram(self, vertex_slice, n_machine_time_steps):
+    def _get_buffered_sdram(
+            self, vertex_slice, n_machine_time_steps, max_rewires_per_ts):
         """
         :param ~pacman.model.graphs.commmon.Slice vertex_slice:
         :param int n_machine_time_steps:
+        :param int max_rewires_per_ts: the max possible rewires per timestep
         :rtype: list(int)
         """
         values = list()
         for variable in itertools.chain(
                 self.__sampling_rates, self.__per_timestep_variables):
             values.append(self.get_buffered_sdram(
-                variable, vertex_slice, n_machine_time_steps))
+                variable, vertex_slice, n_machine_time_steps,
+                max_rewires_per_ts))
         return values
 
     def write_neuron_recording_region(
             self, spec, neuron_recording_region, vertex_slice,
-            data_n_time_steps):
+            data_n_time_steps, max_rewires_per_ts):
```

Comment:
Again, if this is passed in to the constructor it saves odd passing around here...